config/jobs/kubernetes/sig-release/cip/container-image-promoter.yaml

```diff
@@ -0,0 +1,132 @@
+          k8s-staging-coredns/manifest.yaml,/etc/cip-service-account/service-account.json \
+          k8s-staging-csi/manifest.yaml,/etc/cip-service-account/service-account.json
+      # TODO: Uncomment this once
+      # https://github.com/GoogleCloudPlatform/k8s-container-image-promoter/issues/16
+      # lands.
+      #env:
+      #- name: CIP_OPTS
+      #  # Be aggressive about keeping the prod registry clean.
+      #  value: "-delete-extra-tags"
+      imagePullPolicy: Always
+
+presubmits:
+  kubernetes/k8s.io:
+  - name: cip-dryrun
+    decorate: true
+    labels:
+      preset-mount-service-account-key: "true"
+    skip_report: false
+    run_if_changed: ".*/manifest.yaml"
+    # It's OK to run things concurrently, because we're only running things with -dry-run.
+    max_concurrency: 10
+    branches:
+    - ^master$
+    spec:
+      containers:
+      - image: gcr.io/cip-demo-staging/cip@sha256:3823b84b54b2e099f4b6e51e4a6225f04c1bf10c606313fb85a3cfff4df8d318
+        args:
+        - /bin/sh
+        - -c
+        - |
+          # TODO: Make the promoter manifest have per-repository service account
+          # credentials. That way, we can make this script just take a delta of
+          # "which manifest files were changed?" and only run the dry run
+          # against the ones that changed, instead of against all of them ilke
+          # this.
+          k8s.gcr.io/run-cip.sh \
+            /app/cip-docker-image.binary \
+            k8s-staging-cluster-api/manifest.yaml,/etc/cip-service-account/service-account.json \
+            k8s-staging-coredns/manifest.yaml,/etc/cip-service-account/service-account.json \
+            k8s-staging-csi/manifest.yaml,/etc/cip-service-account/service-account.json
+        env:
+        - name: CIP_OPTS
+          # Just perform a dry run.
+          value: "-dry-run"
+
+postsubmits:
+  cip-bot/cip-manifest:
+  - name: cip-prodrun
+    decorate: true
+    labels:
+      preset-mount-service-account-key: "true"
+    run_if_changed: "k8s.gcr.io/k8s-staging-cluster-api/manifest.yaml"
+    skip_report: false
+    # Never run more than 1 job at a time. This is because we don't want to run
+    # into a case where an older manifest PR merge gets run last (after a newer
+    # one).
+    max_concurrency: 1
+    branches:
+    - ^master$
+    spec:
+      containers:
+      - image: gcr.io/cip-demo-staging/cip@sha256:3823b84b54b2e099f4b6e51e4a6225f04c1bf10c606313fb85a3cfff4df8d318
+        args:
+        - /bin/sh
+        - -c
+        - |
+          k8s.gcr.io/run-cip.sh \
+            /app/cip-docker-image.binary \
+            k8s-staging-cluster-api/manifest.yaml,/etc/cip-service-account/service-account.json \
+            k8s-staging-coredns/manifest.yaml,/etc/cip-service-account/service-account.json \
+            k8s-staging-csi/manifest.yaml,/etc/cip-service-account/service-account.json
+        imagePullPolicy: Always
+
+presets:
```

Comment:
Will you just put the `service-account-creds` volume and volume mount into your three pod specs and not use a preset?

Reply:
done